docs/source/using/profiles.adoc

```diff
@@ -8,29 +8,47 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+The profile feature allows creating multiple instances of {project}. 
 
-[[creating-profile]]
+A profile is an instance of the {project} VM along with all of its configuration and state.
+
+Each {project} profile can be created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+Profiles share nothing between them.
+Things like cache, add-ons and persistent config are specific to a single profile.
+This means if you have set some specific configuration parameters for a profile, these settings do not apply to other profiles.
+
+The current profile being used is called the active profile.
+By default {project} commands will run against the active profile.
+You can run the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command to findout the active profile.
+
+The profile feature introduced a global `--profile` flag and xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
+
+The `--profile` global flag is used with other {project} commands to run a command against a specific profile.
```

Comment:
The `--profile` global flag can be specified with any {project} command to run it against the specific profile bypassing the currently active profile.